stories/atoms/components/textField/TextField.stories.mdx

```diff
@@ -266,13 +266,22 @@ The `withBorder` property allows you to add a themed border to the textField and
 
 ## Multiline
 
-The `multiline` property allows you to write text on several lines. The simple text field becomes a text area.  
-When `multiline` is `true` and [disableAreaResize](/docs/atoms-textfield--disable-area-resize) is `false` the [size](/docs/atoms-textfield--sizes) is automatically `auto`.
+The `multiline` property allows you to write text on several lines. The simple text field becomes a text area.
+The [`size`](/docs/atoms-textfield--sizes) is automatically **auto**.  
+When `fullWidth` is **true** the user can only resize the height of the textField.
 
 <Canvas>
   <Story name="Multiline">
-    <div style={{ display: 'flex', justifyContent: 'center', maxWidth: '100%' }}>
-      <TextField placeholder="I'm a multiline text field." multiline size="large" />
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '10px',
+        alignItems: 'center'
+      }}
+    >
+      <TextField placeholder="I'm a multiline text field." multiline />
+      <TextField placeholder="I'm a multiline text field in full width." multiline fullWidth />
```

Comment:
Also, you could add some indications in the TextField placeholder.
You could take advantage of this to write on several lines, and indicate for example:
"I'm a multiline text field: My size is automatically set to auto & you can resize my width and height!
This is an idea, I let you see what you want to do for both TextField examples.